### PR TITLE
chore: Add GitHub CI configuration and format code

### PR DESCRIPTION
**Pull request by hanzeINGH**
- Created .github/workflows directory for CI configuration
- Set up build and test steps in the CI pipeline
- Added code formatting job to ensure consistent style
- Added badge to README.md to display CI status

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, master ]
```
Comment (chyroc) [inline .github/workflows/ci.yml:5]: 去掉 master

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  pull_request:
+    branches: [ main, master ]
```
Comment (chyroc) [inline .github/workflows/ci.yml:7]: name: CI
on:
  push:
    branches:
      - main
  pull_request:
  merge_group:

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
```
Comment (chyroc) [inline .github/workflows/ci.yml:19]: 支持其版本吗

Comment (hanzeINGH) [reply .github/workflows/ci.yml:19]: 国内使用最多的 java 版本就是 8，不过java 一般是向下兼容的，更高版本也可以使用

**Pull request merged**